frontend/src/components/MeetingRoom/VideoTileCanvas/VideoTileCanvas.tsx

```diff
@@ -74,8 +75,12 @@ const VideoTileCanvas = ({
     wrapRef,
   });
 
-  // Height is 100vh - toolbar height (80px) - 24px wrapper margin
-  const heightClass = '@apply h-[calc(100vh_-_104px)]';
+  // Height is 100vh - toolbar height (80px) and header height (80px) - 24px wrapper margin on mobile
+  // Height is 100vh - toolbar height (80px) - 24px wrapper margin on desktop
+  const heightClass = isMobile()
+    ? '@apply h-[calc(100vh_-_184px)]'
+    : '@apply h-[calc(100vh_-_104px)]';
+
```

Comment:
This might be ok for now, however I think we need to maintain the current behaviour where a desktop window squeezed to mobile width shows the same view as mobile.
IMO `isMobile()` should be for stuff like disabling screenshare, rather than layout stuff, which we can do by viewport

Reply:
Please see this: https://github.com/Vonage/vonage-video-react-app/pull/47/files#diff-963f2e2a33b88c0cbb7f06da4a829ece6469980b8f45e7b75196bfe5981df80eR78
I think we need to be consistent in our approach for these mobile features

Reply:
Thanks for bring this up! I was thinking the same thing but didn't want to hold this up for my PR 😅